docs/Using Fleet/downgrading-fleet.md

```diff
@@ -0,0 +1,40 @@
+# Downgrading from Fleet Premium
+
+Follow these steps to downgrade your Fleet instance from Fleet Premium.
+
+> If you'd like to renew your Fleet Premium license key, please contact us [here](https://fleetdm.com/company/contact).
+
+## Back up your users and update all team-level users to global users
+
+1. Run the `fleetctl get user_roles > user_roles.yml` command. Save the `user_roles.yml` file so that you can restore user roles if you decide to upgrade later.
+2. Head to the **Settings > Users** page in the Fleet UI.
+3. For each user that has any team listed under the **Teams** column, select **Actions > Edit**, then select **Global user**, and then **Save**. Delete any users that shouldn't have global access.
+
+## Move all team-level scheduled queries to the global level
+
+1. Head to the **Schedule** page in the Fleet UI.
+2. For each scheduled query that belongs to a team, copy the name in the **Query** column, select **All teams** in the top dropdown, select **Schedule a query**, past the name in the **Select query** field, choose the frequency, and select **Schedule**.
+3. Delete each scheduled query that belongs to a team because they will no longer run on any hosts following the downgrade process.
```

Comment:
These instructions are out-of-date; since we moved the schedule features into queries, downgrading users would need to duplicate their team queries at the global level (we don't have a way to transfer queries between teams or to no team).

I think this should actually look pretty similar to the instructions for policies below.